template.yaml

```diff
@@ -653,6 +653,7 @@ Resources:
     Type: AWS::Serverless::Function
     Properties:
       FunctionName: !Sub ${AWS::StackName}-audit-message-delimiter
+      Timeout: 120
```

Comment:
firehose console was complaining about lambda timeout of <30 sec so i bumped it up

Reply:
since this ingests everything, i thought i would give it 4x increase rather than 2x increase for splunk firehose funcs